todo/src/reducers/reducer.js

```diff
@@ -0,0 +1,28 @@
+import { AddTodo } from "../actions/AddTodo";
+
+const initialState = {
+    todos: [
+      { text: "Wal the Dog",
+        completed: false },
+      { text: "Choir Practice", 
+        completed: false },
+      { text: "Take kids shopping", 
+        completed: false },
+          
+    ]}
+
+
+    function reducer(state = initialState, action) {
```

Comment:
Yup just as we talked about; this function needs to be finished before you build your app so it can return a default initial state